src/BoundingBox.cpp

```diff
@@ -47,7 +47,9 @@ glm::vec3 BoundingBox::calcReflNormal(BoundingBox& other) {
    }
    if((this->min_.y - epsilon <= other.max_.y && this->min_.y + epsilon >= other.max_.y) ||
            (this->max_.y - epsilon <= other.min_.y && this->max_.y + epsilon >= other.min_.y)){
-       normal = glm::vec3(0.0, 1.0, 0.0);
+
+       // TODO(rgarmsen2295): Hack, should be y as normal, figure out why collisions are matching x instead of y only
+       normal = glm::vec3(1.0, 0.0, 0.0);

```

Comment:
Unless you changed something in the JSON version, the hedges aren't rotated, they're just scaled differently (longer in x vs longer in z).  I thought about that as being part of the problem but didn't see anything immediate in the code pointing to an issue.

As far as the cube.obj file itself having an odd initial rotation, would it matter? The min and max are calculated based off the min and max vertices of the object on load (and then recalculated as the object is transformed), so while worth looking into I can't think of anything off the bat.